### DISCUSSION TITLE
`@예약어` 메서드의 네이밍 변경의 건

### DISCUSSION THREAD

**Discussion by BOBx5** [lukewire129/BytePacketSupport#9] [Polls]
* Issue: [[제안] @reservedWord 메서드의 네이밍 변경](https://github.com/lukewire129/BytePacketSupport/issues/8#top)
* Forum: [PacketByte Support 라이브러리](https://forum.dotnetdev.kr/t/packetbyte-support/10306)

위 이슈 및 포럼 토론 내용을 바탕으로 

기존 `Append` 메서드의 네이밍 변경과 관련한

투표를 생성하고자 합니다.

**Comment by lukewire129** [comment]
AppendInt는

![image](https://github.com/lukewire129/BytePacketSupport/assets/54387261/519c5985-b9df-4c77-87aa-5df203f13236)

소문자로 가는건 어떨까싶네요 :)

또한 
AppendUint32의 경우엔

CLR을 을 지향하려면 AppendUInt32가 맞을거같습니다.

**Reply by BOBx5** [reply]
앗 제가 오타를 냈군요

`AppendUInt32` 가 맞습니다.

**Comment by RainyLens** [comment]
흔히 말하는 
[Common Language Specification (CLS)](https://learn.microsoft.com/en-us/dotnet/standard/language-independence)에 맞추는 것이 제일 바람직하지 않을까?라고 생각합니다 ㅎㅎ
Non-Complicant의 경우엔 네이밍 컨벤션만 맞춰주심 되지 않을까 싶어요.

**Reply by lukewire129** [reply]
의견 감사합니다 :)
